website/versioned_docs/version-2.0.0/concepts-inventory.md

```diff
@@ -0,0 +1,59 @@
+---
+id: version-2.0.0-concepts-inventory
+title: Inventory
+original_id: concepts-inventory
+---
+
+The default Reaction system does not have inventory tracking enabled for any products. All products that you publish to the catalog are available for purchase in any quantity and will never show up as low quantity, sold out, or backordered.
+
+If you do track inventory, Reaction provides a [Simple Inventory plugin](./core-plugins-simple-inventory.md) that allows you to update inventory in stock and other inventory settings in the operator UI. It also keeps track of inventory that was ordered but which you have not yet removed from the in-stock number, to ensure that you do not sell what you don't have available.
+
+If you use a third-party inventory system or want to design your own inventory system, you can [create a custom plugin](./how-to-create-inventory-plugin.md).
+
+Regardless of which plugin tracks your inventory, there are some general ways in which inventory impacts your catalog products, catalog variants, and cart items.
+
+## Catalog Product Variant Inventory Data
+
+The following data is available for every variant in the catalog. If a variant has options, then these values are summed or aggregated from the child options.
+
+### Inventory In Stock
+
+The "in stock" number represents how many units you actually have in stock.
+
+### Reserved Inventory
+
+Most inventory plugins will want to track new orders and consider the ordered quantity of each item to be "reserved" until you pull the order and decrease the "in stock" quantity. If an inventory plugin doesn't track this, then "reserved" inventory will always be zero.
+
+### Inventory Available To Sell
+
+The "available to sell" number represents how many units the system will allow a shopper to order. It is calculated by subtracting "reserved" from "in stock". If an inventory plugin doesn't track "reserved", then the available inventory will always be the same as the in stock inventory.
+
+### Can Backorder
+
+Each variant may or may not have backordering enabled. With backordering enabled, the orders service will allow an order to be placed for a quantity higher than the "available to sell" quantity.
```

Comment:
Inconsistant use of hyphens in the word `backordering`. I'm not sure which is correct, but we should stick to one.